GraknConsole.java

```diff
@@ -106,7 +112,7 @@ public boolean runScript(CommandLineOptions options) {
             for (; i < commandStrings.size() && !cancelled[0]; i++) {
                 String commandString = commandStrings.get(i);
                 printer.info("+ " + commandString);
-                ReplCommand command = ReplCommand.getCommand(commandString);
+                ReplCommand command = ReplCommand.getCommand(commandString, client);
```

Comment:
I suppose we don't need to pass in the entire `client` down? Just a boolean down?

Reply:
Updated to `boolean isCluster`